sql-bricks.js

```diff
@@ -41,7 +41,7 @@
 
     var str = this.str;
     if (!opts)
-      opts = default_opts;
+      opts = _.extend({}, default_opts);

```

Comment:
by the way, shouldn't Object.assign() be used instead of all _.clone, _.extend and _.defaults? Or it's still unsupported in older nodejs?